python/sklearnserver/setup.py

```diff
@@ -29,12 +29,13 @@
     description='Model Server implementation for scikit-learn. \
                  Not intended for use outside KFServing Frameworks Images',
     long_description=open('README.md').read(),
-    python_requires='>3.4',
+    python_requires='>3.7',
```

Comment:
I think we are now on Python 3.8 for Hopsworks. Is it ok with Python 3.7?

Reply:
It has to be greater than 3.7, so minimum 3.8.0.